src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
I think it's a good idea to use var modules here and for things like document and documentElement. It doesn't affect the built file – it just becomes a var statement – and it helps us ensure we're using the right document/documentElement/timeout function. Dave suggested this is taken care of by basic unit tests, but I don't think that's what we've seen historically. We've accidentally used the wrong document over and over and over until we did this.